### ISSUE TITLE
Nextra3, can't imported shared .mdx if it has a reference to Callout.

### ISSUE THREAD

**Issue by eat-sleep-code** [shuding/nextra#3821]
In Nextra3, it appears I can no longer do the following: 

**pages/governance/vendor-engagement.mdx:** 

`import DomainRegistrationSnippet from "../../snippets/domain-registration.mdx";`

**snippets/domain-registration.mdx:**

```
import { Callout } from "nextra-theme-docs";

<Callout type="warning" emoji="⚠️">
            Some Text Goes Here
</Callout>

Some more text here...
```

This used to work in Nextra2, but now in Nextra3 I get:

```
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
```

ChatGPT sent me down a rabbit hole of outdated information, so here I am.   This is likely not a bug, but not readily apparent "breaking change documentation".

---

TLDR; How do I reference a shared snippet that has calls to other Nextra components, etc. within Nextra 3?

**Comment by oobg** [comment]
https://the-guild.dev/blog/nextra-3?utm_source=nextra.site&utm_campaign=blog_page&utm_content=blog_link#steps-callout-tabs-cards-and-filetree-exports-were-removed

Please refer to the content at the link above!

In the “Nextra 2 -> 3 Migration Guide,” under item 2, you can see that the import method has changed.

It no longer imports from nextra-theme-docs but instead from nextra/components.

It might be helpful to review the official documentation to confirm how to call it correctly.

https://nextra.site/docs/guide/built-ins/callout#usage

```jsx
import { Callout } from 'nextra/components'
 
<Callout type="info" emoji="ℹ️">
  Today is Friday.
</Callout>
```

**Comment by dimaMachina** [comment]
@oobg thanks!

**Comment by eat-sleep-code** [comment]
Thanks.   I will review this after I "return to the office" in the new year.
